laa_crime_application_store_app/routers/v1/application.py

```diff
@@ -102,4 +107,45 @@ async def put_application(
         return Response(status_code=409)
 
     logger.info("APPLICATION_UPDATED", application_id=application.application_id)
+    NotificationService().notify(db, request, application.application_id)
     return Response(status_code=201)
+
+
+@router.post("/subscriber")
+@auth_logger
+async def post_subscriber(
+    request: Request,
+    subscriber: Subscriber,
+    db: Session = Depends(get_db),
+):
+    logger.info("CREATING_SUBSCRIBER")
+    existing_subscriber = NotificationService.subscribe(
+        db, subscriber.subscriber_type, subscriber.webhook_url
+    )
+
+    if existing_subscriber is None:
```

Comment:
It would make sense to have this follow the logic in the 
[application router](https://github.com/ministryofjustice/laa-crime-application-store/blob/d3c89edf53acc29c4f0f7e938d33f95348ad2d5c/laa_crime_application_store_app/routers/v1/application.py#L75) and the [application service](https://github.com/ministryofjustice/laa-crime-application-store/blob/d3c89edf53acc29c4f0f7e938d33f95348ad2d5c/laa_crime_application_store_app/services/v1/application_service.py#L102)

So you only get `None` if there is a duplicate subscriber found.  Although to be honest that logging info and assumption is not great as duplicate application is not the only reason the db commit would fail but that's a completely separate issue.

Reply:
Ok, have adjusted.